### ISSUE TITLE
returns in quoted fields are not supported

### ISSUE THREAD

**Issue by wvdvegt** [tspence/csharp-csv-reader#53]
I'm trying to read a file with something as this in it (a line break inside a quoted field):

```
"test","
",,,,"Normal","False",,,"Normal",""
```

but the resulting parsed line is terminated early at the field containing the return. So its clearly not supported as far as i can tell.

Point 6 of the definition/standard of csv in https://www.rfc-editor.org/rfc/rfc4180 shows it should be supported (but the field containing the line break needs to be quoted).

**Comment by tspence** [comment]
Thanks very much for this report!

I have had unit tests for embedded line breaks since I first started writing this code library, so I am surprised to see this.  Let me add this as a test case and see if I can spot any issues.

**Comment by tspence** [comment]
Looks like you did find a bug!  Although I had tests for embedded newlines, I did not have a test for an embedded newline as the last character of a text qualified field.  Added that, and will ship a new version with this fix right away!

**Comment by wvdvegt** [comment]
Seems to be an error in my code: I did not correctly specify the field delimiter (wrong type of quote). This morning I noticed it csv-reader also failed on a , inside a quoted fields as well, which was a clear indication something was very wrong so I started digging and ran across an error in my code.

FYI Parsing seems ok with but the main as fix-issue-53 branches. 

It might be a good idea to add the examples from the RFC 4180 (as it is the official specification) as a tests.
